CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+You will find here a summary of the different things you should do / look up to contribute to the project.
+
+## Starting
+
+* First, [fork](https://github.com/ArkScript-lang/Ark/fork) the repository
+* Then, clone your fork: `git clone git@github.com:username/Ark.git`
+* Create a branch for your feature: `git checkout -b feat-my-awesome-idea`
+* When you're done, push it to your fork and submit a pull request!
+
+Don't know what to work on? No worries, we have a [list of things to do](https://github.com/ArkScript-lang/Ark/projects). Also, you can check the issues to find something to do!
+
+## C++ coding guidelines
+
+* Indent with **4 spaces**
+* Every brace (`{`, `}`) must be on its own line
+* Conditions with a single statement (`if (condition) do_this();`) do not need to be enclosed in braces
+* For loops should be optimized whenever optimized, as following:
```

Comment:
Suggested modification:
`For-loops should be optimised whenever possible, as follows:`
- For-loops should have an hyphen to help distinguish between the for-loops and "for loops"
- "optimized whenever optimized" did not make any sort of sense
- "as following" can, and should, be replaced with "as follows"